pulsar-common/src/main/java/org/apache/pulsar/common/naming/Constants.java

```diff
@@ -24,6 +24,11 @@
 public class Constants {
 
     public static final String GLOBAL_CLUSTER = "global";
+    public static final String SCHEMA_REGISTRY_CLASS_NAME;
```

Comment:
nit: please declare the value here: "org.apache.pulsar.broker.service.schema.SchemaRegistryServiceDisabled"